lms/views/admin.py

```diff
@@ -85,6 +85,7 @@ def update_instance(self):
             ("canvas", "sections_enabled"),
             ("canvas", "groups_enabled"),
             ("blackboard", "files_enabled"),
+            ("blackboard", "groups_enabled"),
```

Comment:
Without this the DB changes weren't made. I didn't notice because I hard coded the value while testing the UI.